packages/docs-reanimated/docs/debugging/logger-configuration.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: logger-configuration
+sidebar_label: Logger configuration
+sidebar_position: 2
+---
+
+# Logger configuration
+
+Reanimated shows warnings that indicate potential issues in the user code. These logs can be configured to be more or less verbose. The default configuration is to show all warnings. You can change this behavior by using the `configureReanimatedLogger` function.
+
+## Reference
+
+To configure the Reanimated logger, import `configureReanimatedLogger` from `react-native-reanimated` and call it with the desired configuration.
+
+```js
+import {
+  configureReanimatedLogger,
+  ReanimatedLogLevel,
+} from 'react-native-reanimated';
+
+// This is a default configuration
+configureReanimatedLogger({
+  level: ReanimatedLogLevel.warn,
+  strict: true, // Reanimated runs in strict mode by default
+});
+```
+
+<details>
+<summary>Type definitions</summary>
+
+```typescript
+function configureReanimatedLogger(config: LoggerConfig): void;
+
+type LoggerConfig = {
+  level?: ReanimatedLogLevel;
+  strict?: boolean;
+};
+
+enum ReanimatedLogLevel {
+  warn = 1,
+  error = 2,
+  fatal = 3,
+}
+```
+
+</details>
+
+### Configuration options
+
+#### `level`
+
+A value od the `ReanimatedLogLevel` enum that defines the minimum level of the logs that will be shown.
+
+#### `strict`
+
+A boolean value that enables or disables strict mode. When strict mode is enabled, Reanimated will show more warnings that can help you to catch potential issues in your code.
+
+## Remarks
+
+- The logger configuration is global and affects all Reanimated animations in your app.
+
+- The `configureReanimatedLogger` function should be called before any Reanimated animations are created, e.g. in the root file of your app.
+
+- Each call to `configureReanimatedLogger` overrides the previous configuration.
```

Comment:
What's the point of calling it again if we want it to be called in the App's root?

Reply:
Mhm, I just wanted to let users know that if `configureReanimatedLogger` is called again, it will override the previous configuration. I can agree that this bullet point might be confusing and unnecessary.